packages/in_app_purchase/lib/src/in_app_purchase_connection/google_play_connection.dart

```diff
@@ -34,6 +34,51 @@ class GooglePlayConnection
     return _billingClient.isReady();
   }
 
+  @override
+  Future<QueryPastPurchaseResponse> queryPastPurchases(
+      {String applicationUserName}) async {
+    List<PurchasesResultWrapper> responses = await Future.wait([
+      _billingClient.queryPurchaseHistory(SkuType.inapp),
+      _billingClient.queryPurchaseHistory(SkuType.subs)
+    ]);
+
+    BillingResponse errorInApp = responses.first.responseCode;
```

Comment:
nit: This naming is a little misleading since this may not be an error. How about `responseCode` instead of `error` here and below?